db/schema.rb

```diff
@@ -286,6 +286,11 @@
   end
 
   create_table "operations", id: :uuid, default: -> { "uuid_generate_v4()" }, force: :cascade do |t|
+    t.string "attachment_content_type"
+    t.string "attachment_file_name"
+    t.integer "attachment_file_size"
+    t.string "attachment_filename"
```

Comment:
Do we need two filename columns? If so, can we please rename so the difference is clear?